sources/packages/web/src/services/InstitutionService.ts

```diff
@@ -37,6 +37,7 @@ import {
   ActiveApplicationDataAPIOutDTO,
   ActiveApplicationSummaryAPIOutDTO,
 } from "@/services/http/dto";
+import { ScholasticStandingAPIInDTO } from "./http/dto/ScholasticStanding.dto";
```

Comment:
Please move this import to the above already imported `from "@/services/http/dto"`.